docs/platform/commands/emit.md

```diff
@@ -0,0 +1,49 @@
+<!--
+title: Serverless Framework Commands - Emit
+menuText: emit
+menuOrder: 11
+description: Emit an event to a serverless service
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/platform/commands/emit)
+<!-- DOCS-SITE-LINK:END -->
+
+# Emit
+
+The `emit` command emits an event to a serverless service
+
+```bash
+serverless emit -n my.event -d'{"foo":"bar"}'
```

Comment:
Not sure if this is a valid command since I always add a space when using options.